pydrive2/test/README.rst

```diff
@@ -8,7 +8,9 @@ Run tests locally
     cd pydrive2/test/settings
     mkdir -p local && cp *.yaml local
 
-2. Setup a Google service account for your Google Cloud Project:
+2. Create **credentials** directory in the root directory of the repo.
```

Comment:
let's use inline `code` for path, directory names, etc

Reply:
here and everywhere else

Reply:
does it make sense to actually include it into the mkdir above?

Reply:
will include, shouldn't make a difference

Reply:
can we create all these directories in tests? create empty stubs with `.keepme` in them, add to gitignore?

it's annoying that:

1. we need to do some things in root, some things inside tests
2. that we can't run tests from any location of the project
3. that we need to include a lot of these boilerplate commands to create all these directories - it's error prone, makes it hard to explain, etc

Reply:
makes sense, everything should be addressed in new commits